src/pages/landing-page/LandingPage.tsx

```diff
@@ -1,22 +1,74 @@
 import LandingPageSection from './LandingPageSection'
 import { useBooksQuery } from '@/hooks/queries/useBooks'
 import LoadingSpinner from '@/components/LoadingSpinner'
+import { useEffect, useState } from 'react'
+import { type Book } from '@/hooks/queries/useBooks'
 
 export default function LandingPage() {
   const booksQuery = useBooksQuery({})
+  const sortedBooksQuery = useBooksQuery({ sortByRatingCount: 'desc' })
+  const youngBooksQuery = useBooksQuery({
+    category: 'Young adult fiction',
+    sortByRatingCount: 'desc',
+  })
+  const historyBooksQuery = useBooksQuery({ category: 'History', sortByRatingCount: 'desc' })
+  const biographyBooksQuery = useBooksQuery({
+    category: 'Biography & Autobiography',
+    sortByRatingCount: 'desc',
+  })
+
+  const categoriesToShow = [
+    {
+      key: 'youngBooks',
+      title: 'Young Adult Fiction',
+      dataTestId: 'young-section',
+    },
+    { key: 'historyBooks', title: 'History', dataTestId: 'history-section' },
+    {
+      key: 'biographyBooks',
+      title: 'Biography and Autobiography',
+      dataTestId: 'biography-section',
+    },
+  ]
+
+  const [bookData, setBookData] = useState<{
+    [key: string]: Book[]
+  }>({
+    youngBooks: [] as Book[],
+    historyBooks: [] as Book[],
+    biographyBooks: [] as Book[],
+  })
+
+  useEffect(() => {
+    setBookData({
+      youngBooks: youngBooksQuery.data?.data || [],
+      historyBooks: historyBooksQuery.data?.data || [],
+      biographyBooks: biographyBooksQuery.data?.data || [],
+    })
+  }, [youngBooksQuery.data, historyBooksQuery.data, biographyBooksQuery.data])
```

Comment:
[SUG] I noticed that `useMemo` is similar to `computed` from vue and might be a better fit here

```suggestion
const bookData = useMemo<{ [key: string]: Book[] }>(
    () => ({
      youngBooks: youngBooksQuery.data?.data || [],
      historyBooks: historyBooksQuery.data?.data || [],
      biographyBooks: biographyBooksQuery.data?.data || [],
    }),
    [youngBooksQuery.data, historyBooksQuery.data, biographyBooksQuery.data],
  )
```

Reply:
I like it, I'll apply that change